pyaedt/generic/plot.py

```diff
@@ -38,13 +38,37 @@
         from matplotlib.path import Path
         import matplotlib.pyplot as plt
 
+        rc_params = {
+            "axes.titlesize": 26,  # Use these default settings for Matplotlb axes.
+            "axes.labelsize": 20,  # Apply the settings only in this module.
+            "xtick.labelsize": 18,
+            "ytick.labelsize": 18,
+        }
+
     except ImportError:
         warnings.warn(
             "The Matplotlib module is required to run some functionalities of PostProcess.\n"
             "Install with \n\npip install matplotlib\n\nRequires CPython."
         )
     except Exception:
-        pass
+        warnings.warn("Unknown error while attempting to import matplotlib.")
```

Comment:
```suggestion
        warnings.warn("Unknown error occurred while attempting to import Matplotlib.")
```
String should have a verb (as well as concluding punctuation).